CMakeLists.txt

```diff
@@ -326,14 +326,22 @@ macro(commom_lib_settings lib write)
         PUBLIC # only for basisu_c_binding.
             BASISU_NO_ITERATOR_DEBUG_LEVEL
         )
-        target_sources(
-            ${lib}
-        PRIVATE
-            # The msvs generator automatically sets the needed VCLinker
-            # option when a .def file is seen in sources.
-            lib/internalexport.def
-            $<${write}:lib/internalexport_write.def>
-        )
+        # The def files that we add have a different syntax depending on the ABI
```

Comment:
Please include the comment here about generators setting necessary the linker options when a .def file is in the sources.

Reply:
:+1: